config.py

```diff
@@ -16,3 +16,6 @@
 # This dict holds aspect ratio data for hydrograph rendered in static/js/hydrograph.js. Keys are 'height' and 'width'
 HYDRO_META = {}
 
+
+SPATIAL_REFERENCE_ENDPOINT = 'http://spatialreference.org/ref/epsg/${epsg_code}/proj4/'
+PROJECTION_EPSG_CODE = '2794'
```

Comment:
I would suggest defaulting this to 4326 which is a global projection. Then override it in instance/config.py. Please update the google doc containing the config.py so we all have easy access to the latest config.py that we are testing with.